include/seqan/graph_msa/graph_align_tcoffee_library.h

```diff
@@ -723,11 +761,97 @@ appendSegmentMatches(StringSet<TString, Dependent<TSpec> > const& str,
         TId id2 = positionToId(str, *itPair);
         assignValueById(pairSet, const_cast<TStringSet&>(str), id1);
         assignValueById(pairSet, const_cast<TStringSet&>(str), id2);
-
+        
         _multiLocalAlignment(pairSet, matches, scores, score_type, 4, SmithWatermanClump());
     }
 }
 
+
+//////////////////////////////////////////////////////////////////////////////
+
+template<typename TString, typename TSpec, typename TSize2, typename TSpec2, typename TScore, typename TSegmentMatches, typename TScoreValues>
+inline void
+appendSegmentMatches(StringSet<TString, Dependent<TSpec> > const& str,
+                     String<TSize2, TSpec2> const& pList,
+                     TScore const& score_type,
+                     TSegmentMatches& matches,
+                     TScoreValues& scores,
+                     LocalPairwiseLibrary,
+                     DeepAlignment)
+{
+    typedef StringSet<TString, Dependent<TSpec> > TStringSet;
+    typedef String<TSize2, TSpec2> TPairList;
+    typedef typename Id<TStringSet>::Type TId;
+    typedef typename Iterator<TPairList const, Standard>::Type TPairIter;
+    typedef typename Value<TScoreValues>::Type TScoreValue;
+    typedef typename Size<TStringSet>::Type TSize;
+
+    // Pairwise alignments
+    TPairIter itPair = begin(pList, Standard());
+    TPairIter itPairEnd = end(pList, Standard());
+    for (; itPair != itPairEnd; ++itPair) {
```

Comment:
So you suggest this:
``for (TPairIter itPair = begin(pList, Standard()), itPairEnd = end(pList, Standard()); itPair != itPairEnd; ++itPair)``
?
It looks a bit bulky. Maybe because of that I decided to split it in several lines, not sure. But I should change, right?